plugins/modules/synchronize.py

```diff
@@ -165,7 +165,22 @@
     description:
       - Add a destination to hard link against during the rsync.
     type: list
-    default:
+    elements: str
+  _local_rsync_path:
```

Comment:
I think these are used internally by the action plugin, and should probably not be documented.

Reply:
ok